### PR TITLE
[mlir][sparse] Generates python bindings for SparseTensorTransformOps.

### PR DESCRIPTION
**Pull request by PeimingLiu**

### CODE REVIEW

**Review by ftynse — APPROVED** [review]

**Review by ftynse — COMMENTED** [review]
Can we have a test, just to make sure the extension is loaded correctly?

**Comment by PeimingLiu** [comment]
> Can we have a test, just to make sure the extension is loaded correctly?

Do you have a pointer to a pre-existing test for me?

**Comment by PeimingLiu** [comment]
> > Can we have a test, just to make sure the extension is loaded correctly?
> 
> Do you have a pointer to a pre-existing test for me?

NVM, found them under the `test` folder, will do!

**Review by yinying-lisa-li — APPROVED** [review]

File: mlir/python/mlir/dialects/SparseTensorTransformOps.td [hunk 1]
```diff
@@ -0,0 +1,14 @@
+//===-- SparseTensorTransfromOps.td ------------------------*- tablegen -*-===//
```
Comment (aartbik) [inline mlir/python/mlir/dialects/SparseTensorTransformOps.td:1]: typo: TransformOps

**Pull request merged**